apps/src/music/blockly/blocks/simple2.js

```diff
@@ -0,0 +1,116 @@
+import {BlockTypes} from '../blockTypes';
+import Globals from '../../globals';
+import {DEFAULT_SOUND} from '../constants';
+
+// Examine chain of parents to see if one is 'when_run'.
+const isBlockInsideWhenRun = ctx => {
```

Comment:
NIT: I think ctx is "context," but my mind initially went to "some cool rendering acronym." 🙃 Mind hydrating the abbreviation?

Reply:
In my mind `ctx` is a pretty standard abbreviation, but that might be because it's the standard naming in Go (see https://pkg.go.dev/context).

Reply:
FWIW, I think we just started using `ctx` when we were prototyping and copying code snippets - this could also just be replaced with `block` for clarity (across all Music Lab block generator code)

Reply:
I can update for the `simple2` blocks but will hold off changing this across other blocks for a separate PR.

Reply:
fwiw, we do seem to have precedent for using `ctx` in [`studio/blocks.js`](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/studio/blocks.js).

Reply:
Update made in the follow-up PR.